src/lib/project.ts

```diff
@@ -276,10 +276,42 @@ export class Project implements ProjectInfo {
   }
 
   async getCommit(commit: string): Promise<Commit> {
-    return new Request(
+    const result = await new Request(
       `projects/${this.id}/commits/${commit}`,
       this.seed.addr,
     ).get();
+    result.stats["insertions"] = result.stats["additions"];
+    delete result.stats["additions"];
+    result.diff["added"] = result.diff["created"];
+    delete result.diff["created"];
+
+    for (const kind of ["added", "deleted", "modified"]) {
```

Comment:
We'll add the missing type for `added` and `deleted` here. Was previously done like this: https://github.com/radicle-dev/radicle-interface/pull/515/files#diff-bf65359317875753c55e35ca05da15e82f77f5578edd8907a14aecdd206fdd99R106.